src/diffusers/models/attention.py

```diff
@@ -65,6 +65,7 @@ def __init__(
     ):
         super().__init__()
         self.only_cross_attention = only_cross_attention
+        self.double_self_attention = double_self_attention
```

Comment:
Is it possible to not pass encoder_hidden_states to forward instead of making a new flag?

Reply:
Resolved in https://github.com/huggingface/diffusers/pull/4549/commits/cf522ff0779786818024efd51ccd3dd7c686cddf